doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -2,111 +2,75 @@
 
 .. _pipeline-tutorial:
 
-#######################
-Running the AP pipeline
-#######################
+.. _pipeline-tutorial-gen3:
+
+###############################
+Running the AP pipeline (Gen 3)
+###############################
 
 Setup
 =====
 
 Pick up where you left off in :doc:`Getting Started <getting-started>`.
-This means you already have a repository of ingested DECam data and have setup
-the LSST Science Pipelines stack as well as ``ap_pipe`` and ``ap_association``.
+This means you already have a repository of ingested DECam data and have set up the LSST Science Pipelines stack.
 
-Your directory structure should look something like
+Your repository should have the following collections, which can be checked using ``butler query-collections <repo>``:
 
-.. code-block:: none
+- **calib/DECam**: biases, flats, defects, camera specs, etc.
+- **raw/DECam**: images to be processesd
```

Comment:
There was very recent discussion about switching this to `<instrument>/raw` and `<instrument>/calib`, which I think will happen, suggest updating it here

Reply:
I know it's been discussed for a while, but has anybody actually committed to those being the standard collection names? If it's not going to be changed for a few months, then it would probably be more helpful to give the collection names that the current code produces.

Reply:
Oh, they've *already* changed it! My mistake.

Reply:
No problem, I just saw the recent discussion and didn't follow up on when changes happened or didn't happen. 👍